mmstudio/src/main/java/org/micromanager/acquisition/internal/acqengjcompat/AcqEngJAdapter.java

```diff
@@ -179,33 +177,13 @@ private Datastore runAcquisition(SequenceSettings sequenceSettings) {
          sb.customIntervalsMs(null);
       }
 
-      // Several "translations" have to be made to accommodate the Clojure engine:
```

Comment:
I think the OMETiff saving may rely on the metadata for timeFirst/slicesFirst etc in order to embed in the OME metadata. Do these changes affect the tags ending up in summary metadata? If this changes something, you may see that OME Tiffs cannot be opened by bio formats importer

Reply:
Bit of testing:

AcqEngJ with these lines present:
Order: time slice channel.   Fiji: Incorrect,   BioFormats:  Incorrect
Order slice channel. Fiji: Correct, BioFormats: Correct (but no colors).
Order channel slice.  Fiji: Correct, Bioformats Correct

AcqEngJ without these lines:
Order: time slice channel.   Fiji: Incorrect,   BioFormats:  Incorrect
Order slice channel. Fiji: Correct, BioFormats: Correct (but no colors).
Order channel slice.  Fiji: Correct, Bioformats Correct

I also tested a bit with the old acquisition engine (results were the same).  

I'll put this code back in, but leave a note why.